tools/python/noinput.py

```diff
@@ -100,10 +164,11 @@
 PQR = "ATOM      1  I   ION     1       0.000   0.000  0.000  1.00  3.00"
 
 Python_kb = 1.3806581e-23
-Python_Na = 6.0221367e+23
+Python_Na = 6.0221367e23
```

Comment:
Lots of duplicated code here -- we should (at some point) consolidate some of this.

Reply:
There are MANY places that have copy/paste/modify code that should be refactored.